tests/client_test.py

```diff
@@ -582,12 +582,6 @@ def testCopy(self):
 
         del self.Server['couchdbkit_test']
 
-    def testSetSecurity(self):
-        db = self.Server.create_db('couchdbkit_test')
-        res = db.set_security({"meta": "test"})
-        self.assert_(res['ok'] == True)
-        del self.Server['couchdbkit_test']
```

Comment:
any particular reason this test is being removed?

Reply:
I thikn this was because it doesn't work with couch2 https://github.com/dimagi/couchdbkit/pull/26